ArcTouch.Core/Models/MoviesDetails.cs

```diff
@@ -0,0 +1,148 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace ArcTouch.Core.Models
+{
+    public class MoviesDetails
+    {
+        [JsonProperty("adult")]
+        public bool Adult { get; set; }
+
+        [JsonProperty("backdrop_path")]
+        public string BackdropPath { get; set; }
+
+        [JsonProperty("belongs_to_collection")]
+        public object BelongsToCollection { get; set; }
+
+        [JsonProperty("budget")]
+        public int Budget { get; set; }
+
+        [JsonProperty("genres")]
+        public List<Genre> Genres { get; set; }
+
+        [JsonProperty("homepage")]
+        public string Homepage { get; set; }
+
+        [JsonProperty("id")]
+        public int Id { get; set; }
+
+        [JsonProperty("imdb_id")]
+        public string ImdbId { get; set; }
+
+        [JsonProperty("original_language")]
+        public string OriginalLanguage { get; set; }
+
+        [JsonProperty("original_title")]
+        public string OriginalTitle { get; set; }
+
+        [JsonProperty("overview")]
+        public string Overview { get; set; }
+
+        [JsonProperty("popularity")]
+        public double Popularity { get; set; }
+
+        [JsonProperty("poster_path")]
+        public object PosterPath { get; set; }
+
+        [JsonProperty("production_companies")]
+        public List<ProductionCompany> ProductionCompanies { get; set; }
+
+        [JsonProperty("production_countries")]
+        public List<ProductionCountry> ProductionCountries { get; set; }
+
+        [JsonProperty("release_date")]
+        public string ReleaseDate { get; set; }
+
+        [JsonProperty("revenue")]
+        public int Revenue { get; set; }
+
+        [JsonProperty("runtime")]
+        public int Runtime { get; set; }
+
+        [JsonProperty("spoken_languages")]
+        public List<SpokenLanguage> SpokenLanguages { get; set; }
+
+        [JsonProperty("status")]
+        public string Status { get; set; }
+
+        [JsonProperty("tagline")]
+        public string Tagline { get; set; }
+
+        [JsonProperty("title")]
+        public string Title { get; set; }
+
+        [JsonProperty("video")]
+        public bool Video { get; set; }
+
+        [JsonProperty("vote_average")]
+        public double VoteAverage { get; set; }
+
+        [JsonProperty("vote_count")]
+        public int VoteCount { get; set; }
+
+        public string FullPosterUrl { get => $"https://image.tmdb.org/t/p/w500{PosterPath}"; }
+        public string GenreListed
+        {
+            get
+            {
+                if (Genres != null)
+                {
+                    return String.Join(", ", Genres.Select(x => x.Name));
+                }
+
+                return String.Empty;
+            }
+        }
+        public string FullBackdropPath { get => $"https://image.tmdb.org/t/p/w500{BackdropPath}"; }
+    }
+
+    public class Genre
```

Comment:
For a more clean and organized solution, you should define only one class per file.

Reply:
I agree.